honeybee_energy/simulation/output.py

```diff
@@ -0,0 +1,518 @@
+# coding=utf-8
+"""Object to hold EnergyPlus simulation outputs."""
+from __future__ import division
+
+from ..reader import parse_idf_string
+from ..writer import generate_idf_string
+
+
+class SimulationOutput(object):
+    """Object to hold EnergyPlus simulation outputs.
+
+    Properties:
+        * outputs
+        * reporting_frequency
+        * include_sqlite
+        * include_html
+        * summary_reports
+    """
+    __slots__ = ('_outputs', '_reporting_frequency', '_include_sqlite',
+                 '_include_html', '_summary_reports')
+    REPORTING_FREQUENCIES = ('Annual', 'Monthly', 'Daily', 'Hourly', 'Timestep')
+
+    def __init__(self, outputs=None, reporting_frequency='Hourly', include_sqlite=True,
+                 include_html=False, summary_reports=['AllSummary']):
+        """Initialize SimulationOutput.
+
+        Args:
+            outputs: A list of EnergyPlus output names as strings, which are requested
+                from the simulation. If None, no outputs will be requested.
+                Note that this object does not check whether the outputs exist
```

Comment:
I agree that we don't need this check for now but this might worth opening an issue for later development to check the outputs against the EnergyPlus IDD.

Reply:
Ok, I added the issue: https://github.com/ladybug-tools/honeybee-energy/issues/35